app/assets/stylesheets/core/_variables.scss

```diff
@@ -65,10 +65,10 @@ $twitter:         #00AAF1;
 $reddit:          #FF3A00;
 
 // Type variables
-$baseFontSize: 				16px;
+$baseFontSize: 				14px;
 $baseLineHeight:      20px;
 $baseFontFamily:      'Roboto', Helvetica, Arial, sans-serif;
-$serifFontFamily:     'Gelasio', Georgia, Times, serif;
+$serifFontFamily:     'Noto Serif', Georgia, Times, serif;
```

Comment:
If we're going to change the font then you also need to change/update the google font import in `layouts/application` and remove the Gelasio one.

Reply:
updated google font import https://fonts.google.com/?query=robot&selection.family=Noto+Serif:400,400i,700|PT+Mono|Roboto:400,500,700,700i&selection.subset=cyrillic,cyrillic-ext,greek,greek-ext